core/commands/pin.go

```diff
@@ -43,13 +43,15 @@ type PinOutput struct {
 }
 
 type AddPinOutput struct {
-	Pins     []string
-	Progress int `json:",omitempty"`
+	Pins        []string
+	OngoingPins []string `json:",omitempty"`
```

Comment:
I was not sure about this json tags

Reply:
I'm actually not sure either. I'll look around.